manifests/machineconfig.crd.yaml

```diff
@@ -233,6 +233,9 @@ spec:
           fips:
             description: "fips controls FIPS mode"
             type: boolean
+          realtime:
+            description: "Controls whether we use real time or traditional kernel"
+            type: boolean
```

Comment:
I was influenced by [abhinavdahiya's comment here](https://github.com/openshift/enhancements/pull/124/files#r349364482) about not using booleans. What about something like `kernelControl: "realtime"`, `kernelControl: "core"`?

Reply:
Yeah, valid thing to consider.  There is also `kernel-alt` at least in RHEL contexts and it's not entirely impossible that that'd be something we want to support.

So perhaps:

`kernel: "default"`
`kernel: "realtime"`

Reply:
yeah, sounds reasonable to me to use string instead of boolean here